src/python/dxpy/templating/templates/nextflow/src/nextflow.sh

```diff
@@ -179,23 +150,25 @@ restore_cache_and_history() {
     "describe":{"fields":{"properties":true}},
     "properties":{"session_id":true,
     "preserve_cache":"true",
-    "nextflow_executable":"'$EXECUTABLE_NAME'"}}' | jq -r '.results[].describe.properties.session_id')
+    "nextflow_executable":"'$EXECUTABLE_NAME'"}}')
+
+    [[ -n $PREV_JOB_DESC ]] ||
+      dx-jobutil-report-error "Cannot find any jobs within the last 6 months to resume from. Please provide the exact sessionID for \”resume\” value or run without resume."
```

Comment:
it is `6M in milliseconds: 6 * 1000*60*60*24*30`

Reply:
https://github.com/dnanexus/dx-toolkit/blob/19184cb9a9565ae8bfe462089a9b3287f9547844/src/python/dxpy/utils/__init__.py#L159

Reply:
@commandlinegirl here is how the number is calculated. Since I am using the api call directly I did the conversion. Is there a more readable way to set it?

Reply:
This is actually quite readable to me (since you didn't use the multiplication result), thanks!